firestore-notice-extension/extension.yaml

```diff
@@ -58,6 +58,13 @@ resources:
       runtime: nodejs16
       httpsTrigger: {}
 
+  - name: unAcknowledgeNotice
```

Comment:
Wouldn't the correct spelling be `unacknowledgeNotice`?

Reply:
Updated.